fix: Added copy icons to first-login message

RichardVanLeeuwen opened the pull request:
Goal:
Prevent mistakes when copy/pasting the first-login info from the info box to the input fields.
Allows copying the email and password by clicking the clipboard icon.
~~Up for discussion: also puts the info in the inputs. This behaviour is different from what a user expects based on experience with the web. Is this a good idea? If this, is it still neccesary to copy to the clipboard. Perhaps the clipboard can be a button with the text "copy to input"?~~

before:
<img width="661" height="154" alt="20250714_22h23m46s_grim" src="https://github.com/user-attachments/assets/bcb1788a-f3d0-48dd-a672-610ad68e2d1a" />
after:
<img width="661" height="155" alt="20250714_22h22m20s_grim" src="https://github.com/user-attachments/assets/4474f142-d7f9-4b48-b020-2159022ae62f" />

Fixes #5551

RichardVanLeeuwen committed:
Added copy icons to first-login message
Also inputs the email/password when clicked.

Kuchenpirat commented:
Hi, thanks for this PR. 
While i think this makes sense, we already have logic implemented to copy to the clipboard. 

Take a look at the AppCopyButton for the correct copy symbol. You might even be able to just use the button, don't have design of it on hand currently. If not we have a use-copy composable that includes the required logic to copy text to the clipboard.

RichardVanLeeuwen committed:
Refactored to use the AppCopyButton.

RichardVanLeeuwen commented:
Thanks for looking at this. I refactored to use the AppCopyButton, works perfect!
This is how it looks now:
<img width="660" height="156" alt="20250722_22h02m24s_grim" src="https://github.com/user-attachments/assets/bf6950de-5b21-4ce3-8e82-05894c5fc3c7" />
I chose to move the buttons over to the right, similar to RecipeIngredients does it. The flex broke the whitespace between the bolded part and non-bolded part, the `&nbsp;` is used as non-collapsible whitespace so it look correct again.
Also modified the height of the icons a bit so they align better with the text.

Kuchenpirat committed:
Merge branch 'mealie-next' into mealie-next

Kuchenpirat commented:
This is still marked as draft, anything you still want to update, or is this ready for review?

RichardVanLeeuwen commented:
Sorry forgot I set that due to the questions.

michael-genson committed:
Merge branch 'mealie-next' into mealie-next

michael-genson reviewed (changes requested):
Generally looks good, but I'm not sure we want the copy button to be all the way to the right:
<img width="617" height="119" alt="image" src="https://github.com/user-attachments/assets/3c2a3705-ae7b-49b1-a70e-67daf573efdc" />

I'd like to see them next to the text (still aligned like they currently are). e.g. (using photoshop)
<img width="617" height="119" alt="2025-08-07_10h47_33" src="https://github.com/user-attachments/assets/c88e3a85-e5e4-4bdf-b42d-e2dffcf4243e" />

Or maybe the alignment would look better with how you used to have it (right next to the text, not aligned with each other). Not sure, open for suggestions there

RichardVanLeeuwen commented:
Not aligning to the right was easy enough, even code-wise it's very clean
<img width="668" height="164" alt="20250810_22h20m12s_grim" src="https://github.com/user-attachments/assets/f1257928-f3e7-43e8-9e8e-d7a666b03d03" />
Would you agree with this?

RichardVanLeeuwen committed:
Removed aligning to the right

michael-genson committed:
Merge branch 'mealie-next' into mealie-next

michael-genson reviewed (approved):
Yup looks great, thank you!
